packages/spruce-skill/interface/pages/skill-views/dashboard_location/index.js

```diff
@@ -24,6 +25,8 @@ const EXAMPLE_SUBSCRIPTION = gql`
 	}
 `
 class DashboardLocationPage extends React.Component {
+	_modal = this.props.skill.modal()
```

Comment:
I don't think we've been denoting private properties w/ a leading underscore elsewhere. Let's afterparty tomorrow on convention @brandonminch @gcpantazis